tests/ffi/lib.rs

```diff
@@ -277,6 +277,27 @@ pub mod ffi {
         #[cxx_name = "rAliasedFunction"]
         fn r_aliased_function(x: i32) -> String;
     }
+
+    struct Dag0 {
+        i: i32,
+    }
+
+    struct Dag1 {
+        dag2: Dag2,
+        vec: Vec<Dag3>,
+    }
+
+    struct Dag2 {
+        dag4: Dag4,
+    }
+
+    struct Dag3 {
+        dag1: Dag1,
+    }
+
+    struct Dag4 {
+        dag0: Dag0,
+    }
```

Comment:
Previously:

<pre>
cxxbridge/sources/tests/ffi/lib.rs.cc:698:17: <b>error: field ‘dag2’ has incomplete type ‘tests::Dag2’</b>
  698 |   ::tests::Dag2 dag2;
      |                 ^~~~
cxxbridge/sources/tests/ffi/lib.rs.cc:575:10: <b>note: forward declaration of ‘struct tests::Dag2’</b>
  575 |   struct Dag2;
      |          ^~~~
cxxbridge/sources/tests/ffi/lib.rs.cc:706:17: <b>error: field ‘dag4’ has incomplete type ‘tests::Dag4’</b>
  706 |   ::tests::Dag4 dag4;
      |                 ^~~~
cxxbridge/sources/tests/ffi/lib.rs.cc:577:10: <b>note: forward declaration of ‘struct tests::Dag4’</b>
  577 |   struct Dag4;
      |          ^~~~
</pre>